src/components/Modal/Modal.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { MdClose as CloseIcon } from 'react-icons/md'
+import Title from 'components/Title/Title'
+import Text from 'components/Text/Text'
+import Container from 'components/Container/Container'
+import styles from 'components/Modal/modal.module.scss'
+
+const Modal = ({ item, onCloseModal, ...props }) => {
```

Comment:
`onCloseModal` no necesita por el momento especificar el sujeto del evento, no hay nada más que se pueda cerrar, por lo que se puede poner directamente `onClose`

```suggestion
const Modal = ({ item, onClose, ...props }) => {
```